src/mmw/js/src/modeling/gwlfe/runoff/views.js

```diff
@@ -75,6 +77,10 @@ var ResultView = Marionette.LayoutView.extend({
         }
     },
 
+    onRender() {
```

Comment:
You can run `./scripts/npm.sh run lint` locally to see warnings for this and other things that can make a PR "Unstable" in Jenkins